sdk/iot/hub/samples/CMakeLists.txt

```diff
@@ -118,17 +105,5 @@ add_executable (
 
```

Comment:
Hi ewerton, can you please add the `${sample_link_libraries}` to the line 95 above?

Reply:
Oh, very good catch, thanks @t-meitan-msft !

Reply:
Done.